sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/util/CosmosPagedIterable.java

```diff
@@ -30,8 +30,7 @@
      * @param cosmosPagedFlux the paged flux use as iterable
      */
     public CosmosPagedIterable(CosmosPagedFlux<T> cosmosPagedFlux) {
-        super(cosmosPagedFlux);
-        this.cosmosPagedFlux = cosmosPagedFlux;
+        this(cosmosPagedFlux, 1);
```

Comment:
is the 1 here meant for prefetch?  
by calling this way, it is setting the page size to 1, should we use 100 to match the default?

Reply:
Just kept the existing behavior -
    public ContinuablePagedIterable(ContinuablePagedFlux<C, T, P> pagedFlux) {
        this(pagedFlux, 1);
    }

Reply:
Never mind - good catch :-) - reverted this change. prefetchCount was always 1 - but changing the defaultPageSize of the flux wasn't intentional. Thank you!